lms/resources/lti_launch.py

```diff
@@ -23,6 +23,34 @@ def __init__(self, request):
         self._request = request
         self._authority = self._request.registry.settings["h_authority"]
 
+    def get_or_create_course(self):
```

Comment:
There was some discussion on the previous PR about moving this to the service. IMO this it's justa helper function to make the course creation (and for now, deal with creating both LegacyCourse and Course) on the context of an lti launch (ie using the params from the request etc).